cookbooks/env_vars/libraries/helpers.rb

```diff
@@ -15,7 +15,7 @@ def fetch_environment_variables(app_data)
       metadata = app_data['components'].find {|component| component['key'] == 'app_metadata'}
       return [] unless metadata && metadata['environment_variables']
 
-      variables = metadata['environment_variables'].map do |var_hash|
+      variables = metadata['environment_variables'].reject{|varname| varname[:name] =~ /^EY_/}.map do |var_hash|
```

Comment:
@paulasaurus This helper method could be used by recipes that end up working with EY_ vars. That's why I suggest moving the filter to the spot where we write the env.cloud file.

Reply:
Ah okay @jfuechsl, was not aware this was a shared library, will move it back to the template.